app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -43,6 +43,11 @@ class Bolt_Boltpay_Block_Checkout_Boltpay extends Mage_Checkout_Block_Onepage_Re
      * @var int flag that represents if the capture is automatically done on authentication
      */
     const AUTO_CAPTURE_ENABLED = 1;
+    
+    /**
+     * @var string To receive the API results in the old freegeoip format, we need ipstack Access Key
+     */
+    const IPSTACK_ACCESS_KEY = '18beed19bdef35bf26f49f2e53c63842';
```

Comment:
I'm little uncomfortable putting this in public github repo, since we're charged based on number of request limit. can you add config in admin for this?